docs/user/dql/functions.rst

```diff
@@ -3127,3 +3127,45 @@ Example::
     |----------------+---------------+-----------------+------------------|
     | DATE           | INTEGER       | DATETIME        | STRUCT           |
     +----------------+---------------+-----------------+------------------+
+
+
+WILDCARD_QUERY
+------------
+
+Description
+>>>>>>>>>>>
+
+``wildcard_query(field_expression, query_expression[, option=<option_value>]*)``
+
+The wildcard_query function maps to the wildcard_query query used in search engine, to return the documents that match a provided text with a given field.
```

Comment:
```suggestion
The wildcard_query function maps to the wildcard_query query used in search engine. It returns documents that match provided text in the specified field.
```

Reply:
Fixed in 0b9752ca994a334de8e9bbc6bd7ac49fadd37944